content/en/docs/concepts/architecture/nodes.md

```diff
@@ -392,6 +392,50 @@ Failed pod objects will be preserved until explicitly deleted or [cleaned up by
 This is a change of behavior compared to abrupt node termination.
 {{< /note >}}
 
+## Swap memory management {#swap-memory}
+
+{{< feature-state state="alpha" for_k8s_version="v1.22" >}}
+
+Prior to Kubernetes 1.22, nodes did not support the use of swap memory, and a
+kubelet would fail to start if swap was detected on a node by default. In 1.22
+onwards, swap memory support can be configured on a per-node basis.
+
+To enable swap on a node, the `NodeSwap` feature flag must be set to true on
+the kubelet, and the `--fail-swap-on` command line flag or `failSwapOn`
+[configuration setting] must be set to false.
+
+[configuration setting]: https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration
+
+A user can also optionally configure `memorySwap.swapBehavior` in order to
+configure how the node will use swap memory. For example,
+
+```yaml
+memorySwap:
+  swapBehavior: LimitedSwap
+```
+
+The available configuration options for `SwapBehavior` are:
+
+- Unset: defaults to `LimitedSwap`
+- `LimitedSwap`: Kubernetes workloads are limited in how much swap they can
+  use. Workloads on the node not managed by Kubernetes can still swap.
+- `UnlimitedSwap`: Kubernetes workloads can use as much swap memory as they
+  request, up to the system limit.
+
+The behaviour of the `LimitedSwap` setting depends if the node is running with
+v1 or v2 of control groups (also known as "cgroups"):
+
+|           | `LimitedSwap`                                                                                          | `UnlimitedSwap`                                                             |
+|-----------|--------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| cgroupsv1 | Kubernetes workloads can use any combination of memory and swap, up to the pod's memory limit, if set. | Kubernetes workloads can use unlimited swap memory, up to the system limit. |
+| cgroupsv2 | Kubernetes workloads cannot use swap memory.                                                           | Kubernetes workloads can use unlimited swap memory, up to the system limit. |
```

Comment:
Please consider revise the table into a simple list:

- Markdown tables are very limited in terms of table formatting, e.g. you cannot break lines freely, you cannot embed lists in cells etc.
- The `UnlimitedSwap` behavior has been clarified above, so no need to repeat it here.